DataFormats/Detectors/Common/include/DetectorsCommonDataFormats/EncodedBlocks.h

```diff
@@ -784,7 +784,7 @@ void EncodedBlocks<H, N, W>::encode(const S* const srcBegin, // begin of source
       encoder = encoderLoc.get();
       dictSize = frequencies->size();
     }
-    assert(probabilityBits == encoder->getProbabilityBits());
+    //    assert(probabilityBits == encoder->getProbabilityBits());
     //    const o2::rans::LiteralEncoder64<S> encoder{*frequencies, probabilityBits};
```

Comment:
just remove both lines, they will be never used.

Reply:
fixed